be/src/vec/exprs/vexpr.h

```diff
@@ -108,7 +108,13 @@ class VExpr {
                                           VExpr* parent, int* node_idx, VExpr** root_expr,
                                           VExprContext** ctx);
     const std::vector<VExpr*>& children() const { return _children; }
-    void set_children(std::vector<VExpr*> children) { _children = children; }
+    void set_children(RuntimeState* state, VExprContext* ctx, std::vector<VExpr*> children,
```

Comment:
this function old call in `dfs_peel_conjunct`, not need check `close_old`

Reply:
fixed